#! Param 1
# mass_arm: [5,0,0,0,0,0,
#            0,5,0,0,0,0,
#            0,0,5,0,0,0,
#            0,0,0,5,0,0,
#            0,0,0,0,5,0,
#            0,0,0,0,0,5]
mass_arm: [10,0,0,0,0,0,
           0,10,0,0,0,0,
           0,0,10,0,0,0,
           0,0,0,10,0,0,
           0,0,0,0,10,0,
           0,0,0,0,0,10]
# damping_arm: [5,0,0,0,0,0,
#               0,5,0,0,0,0,
#               0,0,5,0,0,0,
#               0,0,0,5,0,0,
#               0,0,0,0,5,0,
#               0,0,0,0,0,5]
damping_arm: [20,0,0,0,0,0,
              0,20,0,0,0,0,
              0,0,20,0,0,0,
              0,0,0,20,0,0,
              0,0,0,0,20,0,
              0,0,0,0,0,20]
# stiffness_coupling: [10,0,0,0,0,0,
#                      0,10,0,0,0,0,
#                      0,0,10,0,0,0,
#                      0,0,0,10,0,0,
#                      0,0,0,0,10,0,
#                      0,0,0,0,0,10]

# stiffness_coupling: [10,0,0,0,0,0,
#                      0,10,0,0,0,0,
#                      0,0,10,0,0,0,
#                      0,0,0,0,0,0,
#                      0,0,0,0,0,0,
#                      0,0,0,0,0,0]



# stiffness_coupling: [25,0,0,0,0,0,
#                      0,25,0,0,0,0,
#                      0,0,25,0,0,0,
#                      0,0,0,0,0,0,
#                      0,0,0,0,0,0,
#                      0,0,0,0,0,0]





stiffness_coupling: [0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0,
                     0,0,0,0,0,0]

# #! Param 2
# mass_arm: [10,0,0,0,0,0,
#            0,10,0,0,0,0,
#            0,0,10,0,0,0,
#            0,0,0,10,0,0,
#            0,0,0,0,10,0,
#            0,0,0,0,0,10]
# damping_arm: [80,0,0,0,0,0,
#               0,80,0,0,0,0,
#               0,0,80,0,0,0,
#               0,0,0,80,0,0,
#               0,0,0,0,80,0,
#               0,0,0,0,0,80]
# stiffness_coupling: [80,0,0,0,0,0,
#                      0,80,0,0,0,0,
#                      0,0,80,0,0,0,
#                      0,0,0,80,0,0,
#                      0,0,0,0,80,0,
#                      0,0,0,0,0,80]

frequency: 100.0

base_link: "j2s6s300_link_base"
end_link: "j2s6s300_link_6" # "j2s6s300_end_effector"  #"wrist_3_link"

# [position (x, y, z)  orientation (x, y, z, w)]
desired_pose:  [0.0,0.40,0.50,       0.707, -0.707, 0.0, 0.0]

workspace_limits: [-0.50, 0.50, 0.25, 0.80, 0.30, 0.75]

arm_max_vel: 1.5
arm_max_acc: 1.0
platform_max_vel: 0.1
platform_max_acc: 0.2

desired_velocity: [0,0,0,0,0,0]

topic_arm_cmd: "j2s6s300_driver/in/cartesian_velocity"

rotation_ft_end:  [1,0,0,
                   0,1,0,
                   0,0,1]
rotation_gripper_ft: [1,0,0,
                      0,1,0,
                      0,0,1]

p_gripper_ft : [0,0,0.16] # the coordinate of 'the mass center of gripper' in f/t sensor frame
p_tool_ft: [0,0,0.16] #the origin of the tool frame in f/t sensor frame